tsdb/engine/tsm1/engine_test.go

```diff
@@ -17,7 +17,7 @@ import (
 // Ensure an engine containing cached values responds correctly to queries.
 func TestDevEngine_QueryCache_Ascending(t *testing.T) {
 	// Generate temporary file.
-	f, _ := ioutil.TempFile("", "tsm1dev")
+	f, _ := ioutil.TempFile("", "tsm")
```

Comment:
Changing one now. Actually, I think that's a little misleading because I think it's not for the file, but for the directory that the temp file goes into